tonic/models/vic/grid_params.py

```diff
@@ -645,16 +845,17 @@ def _run(args):
 
 # -------------------------------------------------------------------- #
 def make_grid(grid_file, soil_file, snow_file, vegl_file, veg_file,
-              lake_file, nc_file='params.nc', version_in='4.2',
+              lake_file, state_file, nc_file='params.nc',
+              nc_state_file='state.nc', version_in='4.2',
               grid_decimal=4, nlayers=3, snow_bands=5, veg_classes=11,
-              max_roots=3, max_numnod=10, cells=None,
-              organic_fract=False, spatial_frost=False,
+              max_roots=3, max_numnod=10, soil_nodes=3, nfrost=1,
+              cells=None, organic_fract=False, spatial_frost=False,
               spatial_snow=False, july_tavg_supplied=False,
               veglib_fcan=False, veglib_photo=False,
               blowing_snow=False, vegparam_lai=False,
               vegparam_fcan=False, vegparam_albedo=False,
               lai_src='FROM_VEGLIB', fcan_src='FROM_DEFAULT',
-              alb_src='FROM_VEGLIB', lake_profile=False):
+              alb_src='FROM_VEGLIB', lake_profile=False, carbon=False):
     """
     Make grid uses routines from params.py to read standard vic format
```

Comment:
If I understand it correctly, this function currently converts both parameters and states right? Maybe update the description to reflect the change (same for the description at the beginning of this file)?